[debug] [2018-03-27T15:33:33.519Z] ----------------------------------------------------------------------
[debug] [2018-03-27T15:33:33.522Z] Command:       /usr/local/bin/node /usr/local/bin/firebase deploy --only functions:addMessage
[debug] [2018-03-27T15:33:33.522Z] CLI Version:   3.17.7
[debug] [2018-03-27T15:33:33.522Z] Platform:      darwin
[debug] [2018-03-27T15:33:33.522Z] Node Version:  v8.10.0
[debug] [2018-03-27T15:33:33.522Z] Time:          Tue Mar 27 2018 11:33:33 GMT-0400 (EDT)
[debug] [2018-03-27T15:33:33.522Z] ----------------------------------------------------------------------
[debug] 
[debug] [2018-03-27T15:33:33.535Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2018-03-27T15:33:33.535Z] > authorizing via signed-in user
[debug] [2018-03-27T15:33:33.537Z] >>> HTTP REQUEST GET https://admin.firebase.com/v1/projects/auth-f86eb  
  
 Tue Mar 27 2018 11:33:33 GMT-0400 (EDT)
[debug] [2018-03-27T15:33:34.093Z] <<< HTTP RESPONSE 200
[debug] [2018-03-27T15:33:34.093Z] >>> HTTP REQUEST GET https://admin.firebase.com/v1/database/auth-f86eb/tokens  
  
 Tue Mar 27 2018 11:33:34 GMT-0400 (EDT)
[debug] [2018-03-27T15:33:34.459Z] <<< HTTP RESPONSE 200
[info] 
[info] === Deploying to 'auth-f86eb'...
[info] 
[info] i  deploying functions
[info] Running command: npm --prefix "$RESOURCE_DIR" run lint
[info] ✔  functions: Finished running predeploy script.
[debug] [2018-03-27T15:33:36.222Z] > [functions] package.json contents: {
  "name": "functions",
  "description": "Cloud Functions for Firebase",
  "scripts": {
    "lint": "eslint .",
    "serve": "firebase serve --only functions",
    "shell": "firebase experimental:functions:shell",
    "start": "npm run shell",
    "deploy": "firebase deploy --only functions",
    "logs": "firebase functions:log"
  },
  "dependencies": {
    "firebase-admin": "~5.10.0",
    "firebase-functions": "^0.9.0"
  },
  "devDependencies": {
    "eslint": "^4.12.0",
    "eslint-plugin-promise": "^3.6.0"
  },
  "private": true
}
[info] i  functions: ensuring necessary APIs are enabled...
[debug] [2018-03-27T15:33:36.224Z] >>> HTTP REQUEST GET https://servicemanagement.googleapis.com/v1/services/cloudfunctions.googleapis.com/projectSettings/auth-f86eb?view=CONSUMER_VIEW  
  
 Tue Mar 27 2018 11:33:36 GMT-0400 (EDT)
[debug] [2018-03-27T15:33:36.224Z] >>> HTTP REQUEST GET https://servicemanagement.googleapis.com/v1/services/runtimeconfig.googleapis.com/projectSettings/auth-f86eb?view=CONSUMER_VIEW  
  
 Tue Mar 27 2018 11:33:36 GMT-0400 (EDT)
[debug] [2018-03-27T15:33:36.919Z] <<< HTTP RESPONSE 200
[debug] [2018-03-27T15:33:36.926Z] <<< HTTP RESPONSE 200
[info] ✔  functions: all necessary APIs are enabled
[debug] [2018-03-27T15:33:36.927Z] >>> HTTP REQUEST GET https://cloudresourcemanager.googleapis.com/v1/projects/auth-f86eb  
  
 Tue Mar 27 2018 11:33:36 GMT-0400 (EDT)
[debug] [2018-03-27T15:33:37.530Z] <<< HTTP RESPONSE 200
[debug] [2018-03-27T15:33:37.530Z] >>> HTTP REQUEST GET https://mobilesdk-pa.googleapis.com/v1/projects/874574211968:getServerAppConfig  
  
 Tue Mar 27 2018 11:33:37 GMT-0400 (EDT)
[debug] [2018-03-27T15:33:38.144Z] <<< HTTP RESPONSE 200
[info] i  functions: preparing functions directory for uploading...
[debug] [2018-03-27T15:33:38.146Z] >>> HTTP REQUEST GET https://runtimeconfig.googleapis.com/v1beta1/projects/auth-f86eb/configs  
  
 Tue Mar 27 2018 11:33:38 GMT-0400 (EDT)
[debug] [2018-03-27T15:33:38.445Z] <<< HTTP RESPONSE 200
[info] i  functions: packaged functions (53.58 KB) for uploading
[debug] [2018-03-27T15:33:38.770Z] >>> HTTP REQUEST POST https://cloudfunctions.googleapis.com/v1/projects/auth-f86eb/locations/us-central1/functions:generateUploadUrl  
  
 Tue Mar 27 2018 11:33:38 GMT-0400 (EDT)
[debug] [2018-03-27T15:34:08.969Z] <<< HTTP RESPONSE 503
[debug] [2018-03-27T15:34:08.969Z] <<< HTTP RESPONSE BODY {
  "error": {
    "code": 503,
    "message": "The service is currently unavailable.",
    "status": "UNAVAILABLE"
  }
}

[debug] [2018-03-27T15:34:08.971Z] TypeError: Cannot create property 'error' on string '{
  "error": {
    "code": 503,
    "message": "The service is currently unavailable.",
    "status": "UNAVAILABLE"
  }
}
'
    at module.exports (/usr/local/lib/node_modules/firebase-tools/lib/responseToError.js:11:16)
    at Request._callback (/usr/local/lib/node_modules/firebase-tools/lib/api.js:45:25)
    at Request.self.callback (/usr/local/lib/node_modules/firebase-tools/node_modules/request/request.js:186:22)
    at emitTwo (events.js:126:13)
    at Request.emit (events.js:214:7)
    at Request.<anonymous> (/usr/local/lib/node_modules/firebase-tools/node_modules/request/request.js:1163:10)
    at emitOne (events.js:116:13)
    at Request.emit (events.js:211:7)
    at IncomingMessage.<anonymous> (/usr/local/lib/node_modules/firebase-tools/node_modules/request/request.js:1085:12)
    at Object.onceWrapper (events.js:313:30)
[error] 
[error] Error: An unexpected error has occurred.
